tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -429,14 +429,15 @@ class CompletionSuite extends BaseCompletionSuite {
   )
 
   check(
-    "import-star-multi-import",
+    "import-star-multi-import".tag(IgnoreForScala3CompilerPC),
     """
       |import scala.collection.immutable.List.{range => r, *@@}
       |""".stripMargin,
     "",
     compat = Map(
       "3" ->
         """|*
+           |*: scala
```

Comment:
This seems incorrect, error is reported in this case

Reply:
After I changed the logic no error is reported anymore. We should not do workspace/classpath search in this case at all, but again it's a separate issue.

Reply:
Could we create a follow up?

Reply:
https://github.com/scalameta/metals/issues/5991